HPCC-17186 DOCS: Spray queue dropdown

g-pan opened the pull request:
Fix HPCC-17186 DOCS: Spray queue dropdown
Document a new queue select field inside of the spray areas of the landing zone widget.
@JamesDeFabia please review
@miguelvazq please review, verify if this sufficiently covers the new queues

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-17186 DOCS: Spray queue dropdown

Fix HPCC-17186 DOCS: Spray queue dropdown
Document a new queue select field inside of the spray areas of the landing zone widget.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

miguelvazq commented:
@g-pan The queue screenshot is not included here.

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: a96e296f8b7fe588f6d0b800ce9570cb0b045518
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 93.386 seconds |


Install hpccsystems-platform-community_6.3.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |93 |93 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |10 |10 |0 |
|setup (thor) |10 |10 |0 |
|setup (roxie) |10 |10 |0 |
|test (hthor) |722 |722 |0 |
|test (thor) |617 |617 |0 |
|test (roxie) |746 |746 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml:
```diff
@@ -1633,10 +1633,11 @@
-      files on the drop zone display. You can choose to upload, download, or
-      delete any files on the drop zone using the landing zone action buttons.
-      You can also spray files to a cluster from this page.</para>
+      <para>Click on the arrow next to a drop zone container, or folder to
```
comma not needed -- "container or folder"

JamesDeFabia reviewed (changes requested):
The title says Spray queue dropdown, but I don't see that described or depicted.

richardkchapman commented:
@g-pan Back to you

g-pan commented:
@JamesDeFabia please review
@miguelvazq please review

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: ee5504e89cbfbd339ca8d50ce6e401876a28b2e2
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.589 seconds |


Install hpccsystems-platform-community_6.3.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |93 |93 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |10 |10 |0 |
|setup (thor) |10 |10 |0 |
|setup (roxie) |10 |10 |0 |
|test (hthor) |722 |722 |0 |
|test (thor) |617 |617 |0 |
|test (roxie) |746 |746 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 9f261be1cb94b275c831f3792d47c590f70987b3
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.451 seconds |


Install hpccsystems-platform-community_6.3.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |93 |93 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |10 |10 |0 |
|setup (thor) |10 |10 |0 |
|setup (roxie) |10 |10 |0 |
|test (hthor) |722 |722 |0 |
|test (thor) |617 |617 |0 |
|test (roxie) |746 |746 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented:
Looks good from my point of view.  @miguelvazq You get the last word.

miguelvazq reviewed (commented):
Seems that ECLWA412.jpg and ECLWA440.jpg are different themes of ECLWatch. 440 is using a theme that we are going to use in a later release and 412 is a different theme altogether (perhaps a different browser)

richardkchapman commented:
@g-pan Back to you

g-pan committed:
HPCC-17186 DOCS: Spray queue dropdown

Fix HPCC-17186 DOCS: Spray queue dropdown
Document a new queue select field inside of the spray areas of the landing zone widget.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

g-pan commented:
updated. @JamesDeFabia please review @miguelvazq please review

JamesDeFabia commented:
Looks good from my point of view. @miguelvazq  Please give final approval

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 38c5c67d08ea36f8b611fe682927e7dbaf1152de
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 94.102 seconds |


Install hpccsystems-platform-community_6.3.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |93 |93 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |10 |10 |0 |
|setup (thor) |10 |10 |0 |
|setup (roxie) |10 |10 |0 |
|test (hthor) |726 |726 |0 |
|test (thor) |619 |619 |0 |
|test (roxie) |747 |747 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

miguelvazq commented:
@g-pan Not sure if it matter but, those internal IP's are not blurred. Not sure what you do documentation wide.

g-pan committed:
HPCC-17186 DOCS: Spray queue dropdown

Fix HPCC-17186 DOCS: Spray queue dropdown
Document a new queue select field inside of the spray areas of the landing zone widget.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

g-pan commented:
Should be ok, still better safe than sorry. Good catch. Image updated.

miguelvazq commented:
Looks good now.

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: d3c859e9a480256d0a39040b7a98143a047af9bf
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 94.101 seconds |


Install hpccsystems-platform-community_6.3.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |93 |93 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |10 |10 |0 |
|setup (thor) |10 |10 |0 |
|setup (roxie) |10 |10 |0 |
|test (hthor) |726 |726 |0 |
|test (thor) |619 |619 |0 |
|test (roxie) |747 |747 |0 |


HPCC Stop: OK
HPCC Uninstall: OK
